api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -34,12 +36,20 @@
     private static final EmailValidator VALIDATOR_EMAIL = EmailValidator.getInstance();
 
     @NotNull
+    @Autowired(required = false)
```

Comment:
This response is late, but spring auto wiring of dynamically constructed collections is a pretty nice feature.